createTestsFromFeature.js

```diff
@@ -1,17 +1,44 @@
 const { createTestFromScenario } = require("./createTestFromScenario");
+const { proceedCurrentStep, getEnvTags } = require("./tagsHelper");
 
 const createTestsFromFeature = parsedFeature => {
-  // eslint-disable-next-line prefer-arrow-callback
-  describe(parsedFeature.feature.name, function() {
-    const backgroundSection = parsedFeature.feature.children.find(
-      section => section.type === "Background"
-    );
-    const otherSections = parsedFeature.feature.children.filter(
-      section => section.type !== "Background"
-    );
-    otherSections.forEach(section => {
-      createTestFromScenario(section, backgroundSection);
-    });
+  const featureTags = parsedFeature.feature.tags;
+  const hasEnvTags = !!getEnvTags();
+  const hasFeatureTags = featureTags && featureTags.length > 0;
+
+  let featureShouldRun = true;
+  if (hasEnvTags) {
+    if (hasFeatureTags) {
+      featureShouldRun = proceedCurrentStep(featureTags);
```

Comment:
proceedCurrentStep name suggests an action, (proceeding the current step), but it returns a boolean. as with any booleans (function or variables) the name should read like part of a question sentence by starting with "is", "has", "should", and similar. 

if(proceedCurrentStep()) doesn't make sense

if (shouldProceedCurrentStep()) - even though a bit weird, makes all the sense in the world ;-)